koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py

```diff
@@ -52,6 +52,10 @@ def update_aws_summary_tables(self, openshift_provider_uuid, aws_provider_uuid,
                 )
                 return
             report_period = accessor.report_periods_for_provider_uuid(openshift_provider_uuid, start_date)
+            if not report_period:
```

Comment:
We would need this change for all three provider specific functions in here for this to resolve the issue for all three providers. This change needs to be in the Azure and GCP functions as well, lines 163-164 for Azure and 266 and 267 for GCP.

Reply:
Gotcha!